scripts/full_eval.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+import os
+
+import yaml
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from arcsf.data.data_module import BlankQAFormatter, get_data
+from arcsf.eval.evaluate import EvaluateOutputs, Evaluator
+from arcsf.utils import get_model_path
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description=("Runs evaluation, for full models."))
+    parser.add_argument(
+        "experiment_name", type=str, help="Relative path to retain model."
+    )
+
+    args = parser.parse_args()
+    experiment_name = args.experiment_name
+    retain_model_dir = get_model_path(experiment_name, "retain")
+
+    exp_config = yaml.safe_load(open(f"{retain_model_dir}/experiment_config.yaml"))
+
+    full_model_dir = get_model_path(exp_config["full_model_name"], "full")
+    print(f"Full model path: {full_model_dir}")
+
+    # load model from full model directory
+    model = AutoModelForCausalLM.from_pretrained(full_model_dir)
+    tokenizer = AutoTokenizer.from_pretrained(full_model_dir)
+    model.config.pad_token_id = tokenizer.eos_token_id
+
+    # load experiment config from the retain model
+
+    qa_formatter = BlankQAFormatter()
```

Comment:
similarly this will need replacing with the new QAFormatter style (also see `scripts/train.py`)